sdk/eventhub/event-hubs/src/impl/eventHubClient.ts

```diff
@@ -291,7 +291,12 @@ export class EventHubClient {
       options.retryOptions = this._clientOptions.retryOptions;
     }
     throwErrorIfConnectionClosed(this._context);
-    return new EventHubProducer(this.eventHubName, this.endpoint, this._context, options);
+    return new EventHubProducer(
+      this.eventHubName,
+      this.fullyQualifiedNamespace,
```

Comment:
Interesting, I was trying to figure out why this was an okay change to make but apparently we only used this field for tracing?

Reply:
That is correct!